libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -309,7 +309,7 @@ class MapboxNavigation @VisibleForTesting internal constructor(
         incidentsOptions,
         null,
         null,
-        false,
+        navigationOptions.enableSensors,
```

Comment:
@SiarheiFedartsou , I was wondering why NN can't use sensors by default but ignore data from them in case of `FixLocation.isMock == true`?

Reply:
Do you mean to detect that we are in simulation mode via this isMock field? I believe it is possible, but not sure solution would be quick - sensors and locations are quite decoupled from each other in our codebase. More than it it is not clear what to do if e.g. we have no locations received, but sensor data is already available. I'd better add some kind of flag which indicates that SDK is in simulation mode.